ouroboros-consensus-shelley/test/Test/ThreadNet/RealTPraos.hs

```diff
@@ -72,21 +72,27 @@ tests = testGroup "RealTPraos"
     [ testProperty "simple convergence" $
           forAll (SecurityParam <$> elements [5, 10])
             $ \k ->
+          -- TODO it would be good to test this with intermediate values, but at
+          -- the moment the test sometimes fails if we do this.
```

Comment:
How/why does it fail?

Reply:
Not sure. Ideally I will look through it with @nfrisby, but I wanted to get this PR merged first before expanding the scope of the Shelley tests.

Reply:
Oh, yeah, I agree that we should merge this. I was just curious :slightly_smiling_face: